tests/patients/test_patients.py

```diff
@@ -261,6 +261,14 @@ def test_query(app, workspace_generator):
     patients = pk.patients.query(workspace.id, "Test^1")
     assert len(patients) == 1
 
+    # Verify paging
+    for i in range(200):
+        pk.patients.create(workspace.id, "patient" + str(i), "Patient " + str(i))
+
+    query = {'page_size': 200}
+    patients = pk.patients._query(workspace, query)
+    assert len(patients) == 202
```

Comment:
I suggest creating fewer patients, making the page size smaller, and testing that you get back at least 2 full pages plus one partial page of patients.

```suggestion
    # Verify paging
    for i in range(10):
        pk.patients.create(workspace.id, "patient" + str(i), "Patient " + str(i))

    query = {'page_size': 5}
    patients = pk.patients._query(workspace, query)
    assert len(patients) == 12
```